src/tests/class_tests/openms/source/NASequence_test.cpp

```diff
@@ -289,6 +289,9 @@ START_SECTION((double getMonoWeight(NASequence::NASFragmentType type = NASequenc
   TEST_REAL_SIMILAR(seq.getMonoWeight(NASequence::YIon, -1), 972.177);
   TEST_REAL_SIMILAR(seq.getMonoWeight(NASequence::DIon, -1), 1034.133);
   TEST_REAL_SIMILAR(seq.getMonoWeight(NASequence::AminusB, -2), 802.117);
+  NASequence seq_notSym = NASequence::fromString("GAU");
```

Comment:
Just pointing this out because it's a general issue: OpenMS uses "camelCase" for function names, but "snake_case" for variable names. Could you rename this to "seq_not_sym"?